docs/resources/fgs_function.md

```diff
@@ -352,6 +363,45 @@ The `aliases` block supports:
 
 * `description` - (Optional, String) Specifies the description of the version alias.
 
+<a name="functiongraph_tactics_config"></a>
+The `tactics_config` block supports:
+
+* `cron_configs` - (Optional, List) Specifies the lists of scheduled policy configurations.
+  The [object](#functiongraph_cron_configs) structure is documented below.
+
+* `metric_configs` - (Optional, List) Specifies the lists of metric policy configurations.
+  The [object](#functiongraph_metric_configs) structure is documented below.
+
+<a name="functiongraph_cron_configs"></a>
+The `cron_configs` block supports:
+
+* `name` - (Required, String) Specifies the name of scheduled policy configuration.
```

Comment:
Add supported character types and length limits for name.

Reply:
fixed